components/Footer/Footer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { FooterColumnsBuilder } from "./FooterColumn";
+import prisma, { Prisma } from "@/lib/prisma";
+import Image from "next/image";
+import { ImageComponent } from "../Shared/ImageComponent";
+import { getAssetPath } from "@/lib/getAssetInfo";
+import { WYSIWYG } from "../Shared/WYSIWYGComponent";
+import style from "./Footer.module.css";
+
+export type FooterProps = Prisma.FooterGetPayload<{}> & {
+  footerColumnsSlot: React.ReactNode;
+};
+
+export async function Footer({
+  logo,
+  content,
+  footerColumnsSlot,
+}: FooterProps) {
+  return (
+    <footer>
+      <div>{footerColumnsSlot}</div>
+      {logo && <ImageComponent path={await getAssetPath(logo)} title="Logo" />}
+      {content && <WYSIWYG style={style} content={content} />}
+    </footer>
+  );
+}
+
+export async function FooterBuilder() {
+  let footer = await prisma.footer.findFirst();
+  if (footer === null) return null;
+  return (
+    <Footer
+      {...footer}
+      footerColumnsSlot={
+        <FooterColumnsBuilder id={footer.id} key={footer.id} />
```

Comment:
The `key` prop is for mapping to components so when eg. the order changes it doesn't have to rerender each component. Here it is not needed since we're not mapping over anything.